data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -184,6 +183,7 @@
     (RoundPpiValuesToNearestInteger,),
     (UpdateFamilyHistoryCodes,),
     (ConvertPrePostCoordinatedConcepts,),
+    (CleanSmokingPpi,),
```

Comment:
These concepts are not post-coordinated at this time, but they are on the list to be updated by Odysseus because they have incorrect information in Athena. These might be made to use post-coordination at that time. I believe there is something more going on with these concepts than the comments on the CR suggest and that the mapping we have defined here should not be changed with post-coordination CR if the concepts are eventually updated.

Reply:
This info sounds like something we should note in Jira, in the class, and potentially here as well.  This doesn't need a long comment, but maybe something hinting that Post Coordination may have unanticipated impacts and we may need to revisit this rule once that is complete.